cli/command/container/opts.go

```diff
@@ -236,9 +236,12 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 	flags.Int64Var(&copts.cpuPeriod, "cpu-period", 0, "Limit CPU CFS (Completely Fair Scheduler) period")
 	flags.Int64Var(&copts.cpuQuota, "cpu-quota", 0, "Limit CPU CFS (Completely Fair Scheduler) quota")
 	flags.Int64Var(&copts.cpuRealtimePeriod, "cpu-rt-period", 0, "Limit CPU real-time period in microseconds")
+	flags.SetAnnotation("cpu-rt-period", "version", []string{"1.25"})
```

Comment:
We'll have to fork `pflag` to make that kind of change, but that's fine with me. Having the flag Vars return the object would be a good idea.

However many of these lines are already too long, so I don't know that adding more to the line is an improvement.